src/data/actions/loginForm.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
```

Comment:
nit: I think it's a little cleaner if the functions (e.g., `setLoginPending`, `setLoginSuccess`, `seterror`) are more in line with the action types (e.g., `LOGIN_REQUEST`, `LOGIN_SUCCESS`, `LOGIN_FAILURE`) kind of like how we're doing it in `actions/courseEnrollments.js`.  Maybe `fetchLoginRequest`, `fetchLoginSuccess`, `fetchLoginFailure`.

nit: You could also rename the actions from `LOGIN_REQUEST` -> `FETCH_LOGIN_REQUEST` to be consistent with the other action names in the app.